src/DocumentDB.ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor.csproj

```diff
@@ -62,11 +62,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="1.9.1" />
+    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.0.0" />
```

Comment:
Do we want to also bump the `<Version>2.0.6</Version>` ?

Reply:
For CFP version I think we should bump the minor version, not revision (like CFP 2.1.0) so that if we need we can continue supporting CFP 2.0.x for customers who can't upgrade to 2.0 SDK. Note that SDK 3.0 is coming at some point as well.

Reply:
(Y) sure .. forgot

Reply:
done